ts/Dashboards/EditMode/Sidebar.ts

```diff
@@ -220,14 +221,32 @@ class Sidebar {
                 sidebar: Sidebar,
                 dropContext: Cell | Row
             ): Cell|void {
+                const headers = ['Apples', 'Pears', 'Plums'];
```

Comment:
How about moving this whole thing into a function like `generateDataTable`?
It has nothing to do with the adding mechanism- it's just a placeholder for the data.

Reply:
This is just a placehodler, which will be changed later. I don't see a reason why it should be moved elsewhere, if it will be removed soon.